x-pack/solutions/security/plugins/security_solution/public/asset_inventory/routes.tsx

```diff
@@ -59,7 +50,7 @@ export const AssetInventoryRoutes = () => {
             <SecuritySolutionPageWrapper noPadding>
               <Suspense fallback={<EuiLoadingSpinner />}>
                 <AllAssetsLazy
-                  rows={rows}
+                  rows={[]}
```

Comment:
maybe we can either remove completely or the other way around, keep as is but pass a local feature flag for development purposes. for cases you want fix/change something quickly without es archiver

```ts
rows={enableMockData ? rows : []}
```

Reply:
I guess that won't be necessary as we have prioritized tickets https://github.com/elastic/security-team/issues/11592 and https://github.com/elastic/security-team/issues/11685 to enable the use of Reindex data and ESArchiver instead of static data for 9.1, but that would be a suggestion if we had the need to keep static data for more time